app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -95,6 +64,23 @@ def verify_permissions(host, template_invocation)
 
         true
       end
+
+      def save_template_invocation(host, template_invocation)
+        template_invocation.host_id = host.id
+        template_invocation.run_host_job_task_id = task.id
+        template_invocation.save!
+      end
+
+      def validate_template_invocation(template_invocation)
```

Comment:
It would be nice if methods which can `raise` would have an exclamation mark in their name